<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task_Dialog</class>
 <widget class="QDialog" name="Task_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameabel">
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="wcetLabel">
         <property name="text">
          <string>WCET</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="wcetEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="deadtimeLabel">
         <property name="text">
          <string>Deadlne</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="deadtimeEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="periodLabel">
         <property name="text">
          <string>Period</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="periodEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="jitterLabel">
         <property name="text">
          <string>Jitter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="jitterEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="offsetLabel">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="offsetEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="isLabel">
         <property name="text">
          <string>isPeriodic(uncheck for sporadic)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="OkbuttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
